views/editChoral.pug

```diff
@@ -7,33 +8,24 @@ block content
       b
         i.fa.fa-dashboard
         |  Edit Choral
-  form.w3-container(method='POST' action='/chorals/edit/'+choralEdit.choralId id='mainForm').new-choral-form
+  form.w3-container(method='POST' action='/chorals/edit/'+ choralEdit.choralId id='mainForm').new-choral-form
```

Comment:
I was also wondering about that. I think it would be a bit cleaner to do it your way but the infrastructure was already there do do it with a url parameter.